scripts/accounts.py

```diff
@@ -0,0 +1,79 @@
+import dbs
+import bson
+
+def convert_accounts():
+    golos_accounts = dbs.golos_db['account_object']
+    cyberway_accounts = dbs.cyberway_db['account']
+
+    id_count = cyberway_accounts.find().sort('id', -1).limit(1)[0]['id'] + 1
+
+    for doc in golos_accounts.find():
+        if cyberway_accounts.find({"name":doc["name"]}).count():
+            continue
+        account = {
+            "id" : bson.Int64(id_count),
+            "name" : doc["name"],
```

Comment:
Not all Golos account names can be converted to Cyberway account (https://github.com/GolosChain/cyberway/issues/31). Required to filter names that cannot be converted to Cyberway names. Cyberway names must satisfies next condition:
* can contain only [.a-z1-5] symbols
* 12,5 length (last symbol can be [.a-o])